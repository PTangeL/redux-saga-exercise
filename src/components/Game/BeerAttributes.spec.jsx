import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

import BeerAttributes from './BeerAttributes'

describe('<BeerAttributes />', () => {
  it('should render component with props', () => {
    const beer = {"id":"82","title":"Karmeliet Triple","description":"<p>Do we really have to introduce Triple Karmeliet, our best-selling beer?</p>","country":"Belgium","type_beer":"Tripel","company":"Brasserie Bosteels","colour":"Blond","graduation":"6 %","size":"33 cl","price":"2.5","photo_link":"https://img.saveur-biere.com/img/p/82-14662-thickbox_sb3.jpg"}
    const wrapper = shallow(<BeerAttributes beer={beer} />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
