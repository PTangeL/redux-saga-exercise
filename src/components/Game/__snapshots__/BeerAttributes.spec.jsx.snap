// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BeerAttributes /> should render component with props 1`] = `
<ul
  className="beer_details"
>
  <li>
    <span>
      Belgium
    </span>
     Country
  </li>
  <li>
    <span>
      Tripel
    </span>
     Style
  </li>
  <li>
    <span>
      Blond
    </span>
     Colour
  </li>
  <li>
    <span>
      33 
    </span>
     cl
  </li>
  <li>
    <span>
      6 %
    </span>
     °Alcohol
  </li>
  <li>
    <span>
      2,50
    </span>
     
    <span>
      £
    </span>
  </li>
</ul>
`;
